### PR TITLE
Updated play services to 9.2.0

### PR DESCRIPTION
**Pull request by MarsVard**

### CODE REVIEW

**Comment by amayatsky** [comment]
Any chance we can have this update?

**Comment by medihack** [comment]
+1

**Comment by fstephany** [comment]
:+1:

**Comment by medihack** [comment]
Unfortunately this pull request is a bit half baked. Why is there a `REACTIVE_LOCATION_GMS_API_KEY` added to the `gradle.properties` (in one commit it was added, then deleted and then added again).
Also the README must be updated.

**Comment by GokhanArik** [comment]
Yes,  we definitely need that for Firebase

**Comment by andretietz** [comment]
+1

**Comment by alorma** [comment]
👍

**Comment by Rainer-Lang** [comment]
+1

**Comment by MarsVard** [comment]
Sorry guys I did some dirty work here, don't have time to redo it. If somebody wants to make a clean one just look at my commits.

File: android-reactive-location/build.gradle [hunk 1]
```diff
@@ -19,7 +19,9 @@ android {
 //TODO: local maven deployment
 
 dependencies {
-    compile 'com.google.android.gms:play-services-location:8.4.0'
+    compile 'com.google.android.gms:play-services-location:9.2.0'
```
Comment (spirosoik) [inline android-reactive-location/build.gradle]: update all to `9.4.0`

**Comment by sergiomarquesmoura** [comment]
any update on this? really hoping to get it

**Comment by alorma** [comment]
Meanwhile you can use this one:

https://github.com/SchibstedSpain/Android-ReactiveLocation

It shows play services 8.4.0 on Readme, but it's compiled with 9.4.0

**Comment by mcharmas** [comment]
Meanwhile you can use version 9.4.0. From my testing library works with newest GMS as it is stated in readme.

```
compile 'com.google.android.gms:play-services-location:8.4.0' //you can use newer GMS version if you need
```

Sorry guys for such delays.

**Comment by mcharmas** [comment]
Updated to 9.4 version. Should be available in few hours

**Comment by nizamsp** [comment]
I am using the following dependencies but I am still getting the proguard warnings and errors.

Warning:pl.charmas.android.reactivelocation.observables.geocode.GeocodeObservable: can't find referenced class com.google.android.gms.maps.model.LatLngBounds
Warning:pl.charmas.android.reactivelocation.observables.geocode.GeocodeObservable: can't find referenced class com.google.android.gms.maps.model.LatLng
Warning:pl.charmas.android.reactivelocation.observables.geocode.GeocodeObservable: can't find referenced class com.google.android.gms.maps.model.LatLngBounds
Warning:pl.charmas.android.reactivelocation.observables.geocode.GeocodeObservable: can't find referenced class com.google.android.gms.maps.model.LatLng
Warning:pl.charmas.android.reactivelocation.observables.geocode.GeocodeObservable: can't find referenced class com.google.android.gms.maps.model.LatLngBounds
Warning:there were 114 unresolved references to classes or interfaces.

Error:Execution failed for task ':wimt:transformClassesAndResourcesWithProguardForRelease'.
> java.io.IOException: Please correct the above warnings first.

dependencies {
    ...
    compile 'pl.charmas.android:android-reactive-location:0.10@aar'
    compile 'com.google.android.gms:play-services-location:9.4.0' //you can use newer GMS version if you need
    compile 'com.google.android.gms:play-services-places:9.4.0'
    compile 'io.reactivex:rxjava:1.2.0' //you can override RxJava version if you need
}

Am I missing something?

**Pull request closed without merging**